build

```
@@ -133,7 +133,7 @@ sucrase('./lib', './.lib-dist');
 
 if (sucrase('./server', './.server-dist')) {
 	replace('.server-dist', [
-		{regex: /(require\(.*?)(lib|sim)/g, replace: `$1.$2-dist`},
+		{regex: /(require\(.*?)(lib|sim|data)/g, replace: `$1.$2-dist`},
```

```suggestion
		{regex: /(require\(.*?)(data|lib|sim)/g, replace: `$1.$2-dist`},
```
This is a minor nit but I like alphabetization

---

This probably isn't even needed and is a remnant from early development, btw